website/docs/r/kms_key.html.markdown

```diff
@@ -12,6 +12,8 @@ This resource can be used for management of keys in both Key Protect and Hyper P
 
 After creating an  Hyper Protect Crypto Service instance you need to initialize the instance properly with the crypto units, in order to create, or manage Hyper Protect Crypto Service keys. For more information, about how to initialize the Hyper Protect Crypto Service instance, see [Initialize Hyper Protect Crypto](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-initialize-hsm) only for HPCS instance.
 
+  ~>**Important:**
+  If the key is used with other IBM Cloud resources that require an `ibm_iam_authorization_policy` resource (requires [service authorization](https://cloud.ibm.com/docs/account?topic=account-serviceauth&interface=ui)), make sure to include `depends_on` targing the `ibm_iam_authorization_policy` involved to ensure proper deletion of resources with `terraform destroy`.
```

Comment:
targing -> targeting

Reply:
Might be good to clarify whether the depends on should be establish from iam to policy to key or kms instance. 
If possible, maybe an example

Reply:
Updated spelling and added link to new example usage section for S2S policy setup